examples/AdminClient/Program.cs

```diff
@@ -583,27 +583,49 @@ static async Task ListConsumerGroupsAsync(string bootstrapServers, string[] comm
         {
             var timeout = TimeSpan.FromSeconds(30);
             var statesList = new List<ConsumerGroupState>();
-            try
+            var groupTypesList = new List<ConsumerGroupType>();
+            var isType = false;
+            var isState = false;
+            for (int i = 0; i < commandArgs.Length; i++)
```

Comment:
Use `foreach` like `foreach (var t in commandArgs)`

Reply:
ok